modules/build.gradle

```diff
@@ -119,6 +119,36 @@ if (workspacesYarnLockFile.exists()) {
 	portalYarnFormat.finalizedBy portalWorkspacesYarnFormat
 }
 
+File playwrightPackageLockJsonFile = new File(rootDir.parentFile, "modules/test/playwright/package-lock.json")
```

Comment:
There may be a problem with this approach (or not, it depends).

Usually the file is not created after an `ant all`, because `ant all` deletes it. So, if someone modifies the tests locally and then, for whatever reason, runs `ant all` or cleans the `package-lock.json` file, the SF will refuse to run.

On the other hand, I understand that unless the file exists the SF cannot run because the JavaScript tooling won't be available so I wonder if it wouldn't be better to force an `npm install` from SF :thinking: 

It could be too much because most of the time there will be nothing to format but on the other hand, if there are modifications, for example, it would be great if we could make the SF run no matter if the use has run `npm install` or the SF does it.

Reply:
Same reasoning applies to workspaces maybe...